test/driver.spec.js

```diff
@@ -57,8 +57,10 @@ describe('did:key method driver', () => {
     });
 
     it('should get the DID Doc in 2018 mode', async () => {
-      const didKeyDriver2018 = driver({
-        verificationSuite: Ed25519VerificationKey2018
+      const didKeyDriver2018 = driver();
+      didKeyDriver2018.registerKeyTypeHandler({
+        multikeyHeaders: ['z6Mk', 'B12N'],
```

Comment:
What is `B12N`? Have you checked what that maps to? Is it a header or is it raw key bytes? I think we might need to adapt keypairs that use `publicKeyBase58` because they probably lack a multikey header.

Reply:
It's clear now that `publicKeyBase58` doesn't include a multikey header, so only `z6Mk` is relevant here. It does look like the Ed25519VerificationKey2018 can be converted from a "fingerprint" (which is a `publicKeyMultibase` value) by using `fromFingerprint` just fine, in theory.

Reply:
Removed `B12N` [6f4b489](https://github.com/digitalbazaar/did-method-key/pull/55/commits/6f4b4897a04b399f1135db110cc90d526260a5c4)